docs/notes/feature-liburl-cookies.md

```diff
@@ -0,0 +1,7 @@
+---
+version: 8.2
+---
```

Comment:
Please delete this header; the explicit `version` header should only be added when the release notes builder can't get the version right by examining the git history.  I added this to allow me to "fix up" release notes after the first DP they appeared in.